charts/flyte/values-gcp.yaml

```diff
@@ -18,31 +201,265 @@ storage:
+  bucketName: "{{ .Values.userSettings.bucketName }}"
   gcs:
-    projectId: <GOOGLE_PROJECT_ID>
+    projectId: "{{ .Values.userSettings.googleProjectId }}"
+
+db:
+  database:
+    port: 5432
+    # -- Create a user called flyteadmin
+    username: flyteadmin
+    host: "{{ .Values.userSettings.dbHost }}"
+    # -- Create a DB called flyteadmin (OR change the name here)
+    dbname: flyteadmin
+    passwordPath: /etc/db/pass.txt
 
 #
 # CONFIGMAPS
 #
 
 configmap:
-  remoteData:
-    remoteData:
-      scheme: "gcp"
-
-  tasks:
-    max-plugin-phase-versions: 1000000
-    task-plugins:
-      enabled-plugins:
-        - container
-        - sidecar
-        - spark
-        - k8s-array
-        - pytorch
-      default-for-task-types:
-        container: container
-        sidecar: sidecar
-        spark: spark
-        container_array: k8s-array
-        pytorch: pytorch
+  adminServer:
+    server:
+      httpPort: 8088
+      grpcPort: 8089
+      security:
+        secure: false
+        useAuth: false
+        allowCors: true
+        allowedOrigins:
+          # Accepting all domains for Sandbox installation
+          - "*"
+        allowedHeaders:
+          - "Content-Type"
+
+  task_resource_defaults:
+    task_resources:
+      defaults:
+        cpu: 1000m
+        memory: 1000Mi
+        storage: 1000Mi
+      limits:
+        storage: 2000Mi
+
+  core:
+    propeller:
+      rawoutput-prefix: "gs://{{ .Values.userSettings.bucketName }}/"
+      workers: 40
+      gc-interval: 12h
+      max-workflow-retries: 50
+      kube-client-config:
+        qps: 100
+        burst: 25
+        timeout: 30s
+      queue:
+        sub-queue:
+          type: bucket
+          rate: 100
+          capacity: 1000
+
+  enabled_plugins:
+    # -- Tasks specific configuration [structure](https://pkg.go.dev/github.com/flyteorg/flytepropeller/pkg/controller/nodes/task/config#GetConfig)
+    tasks:
+      # -- Plugins configuration, [structure](https://pkg.go.dev/github.com/flyteorg/flytepropeller/pkg/controller/nodes/task/config#TaskPluginConfig)
+      task-plugins:
+        # -- [Enabled Plugins](https://pkg.go.dev/github.com/lyft/flyteplugins/go/tasks/config#Config). Enable sagemaker*, athena if you install the backend
+        # plugins
+        enabled-plugins:
+          - container
+          - sidecar
+          - spark
+          - k8s-array
+          - pytorch
+        default-for-task-types:
+          container: container
+          sidecar: sidecar
+          spark: spark
+          container_array: k8s-array
+          pytorch: pytorch
+
+  logger:
+    logger:
+      level: 5
+
+  # -- Section that configures how the Task logs are displayed on the UI. This has to be changed based on your actual logging provider.
+  # Refer to [structure](https://pkg.go.dev/github.com/lyft/flyteplugins/go/tasks/logs#LogConfig) to understand how to configure various
+  # logging engines
+  # TODO how does this work with GCS logging?
```

Comment:
I believe we want stackdriver logs here. Something like this perhaps:
```
plugins:
  logs:
    # Log links can link to multiple options
    # #1 Kubernetes dashboard is disabled in GCP
    kubernetes-enabled: false
    # #2 GCP stackdriver
    stackdriver-enabled: true
    stackdriver-logresourcename: k8s_container
  k8s-array:
    logs:
      config:
        stackdriver-enabled: true
        stackdriver-logresourcename: k8s_container
```

Reply:
done